test/system/700-play.bats

```diff
@@ -196,6 +196,30 @@ EOF
     run_podman rm -a
 }
 
+@test "podman kube play read-only" {
+    YAML=$PODMAN_TMPDIR/test.yml
+    run_podman create --pod new:pod1 --name test1 $IMAGE touch /test
+    run_podman create --pod pod1 --read-only --name test2 $IMAGE touch /test
+    run_podman create --pod pod1 --read-only --name test3 $IMAGE touch /tmp/test
+    run_podman kube generate pod1 -f $YAML
+
+    run_podman pod rm -a
+    run_podman rm -a
```

Comment:
We can remove the two commands by adding `--replace` to `kube play` below.

Reply:
I just remove the stray images, to remove warnings on cleanup.